integreat_cms/cms/views/pages/page_form_view.py

```diff
@@ -89,103 +88,27 @@ def get(self, request: HttpRequest, *args: Any, **kwargs: Any) -> HttpResponse:
-                            },
-                        )
-                    messages.warning(request, status_message)
-                # Show information if a public translation exists even if the latest version is not public
-                if public_translation := page.get_public_translation(language.slug):
-                    if page_translation.status == status.DRAFT:
-                        messages.warning(
-                            request,
-                            _("The latest changes have only been saved as a draft."),
-                        )
-                    revision_url = reverse(
-                        "page_versions",
-                        kwargs={
-                            "region_slug": region.slug,
-                            "language_slug": language.slug,
-                            "page_id": page.id,
-                            "selected_version": public_translation.version,
-                        },
-                    )
-                    message = _(
-                        "Currently, <a>version {}</a> of this page is displayed in the app.",
-                    ).format(public_translation.version)
-                    messages.info(
-                        request,
-                        translate_link(
-                            message,
-                            attributes={
-                                "href": revision_url,
-                                "class": "underline hover:no-underline",
-                            },
-                        ),
-                    )
+        disabled = self.get_initial_disable_state(request, page)
 
-        # Make form disabled if user has no permission to edit the page
-        if not request.user.has_perm("cms.change_page_object", page):
-            disabled = True
-            messages.warning(
+        if page and page_translation and page_translation.status != status.PUBLIC:
+            self.indicate_autosave_or_pending_review(
                 request,
-                _("You don't have the permission to edit this page."),
+                page=page,
+                page_translation=page_translation,
+                region=region,
+                language=language,
             )
-        # Show warning if user has no permission to publish the page
-        elif not request.user.has_perm("cms.publish_page_object", page):
-            messages.warning(
+
+            # Show information if a public translation exists even if the latest version is not public
+            self.handle_public_version_not_latest(
                 request,
-                _(
-                    "You don't have the permission to publish this page, but you can propose changes and submit them for review instead.",
-                ),
+                page=page,
+                page_translation=page_translation,
+                language=language,
             )
 
+        disabled = self.handle_permissions(request, page)
```

Comment:
So ``handle_permissions`` returns ``True`` if the user does not have permissions to edit the page? I find that a bit confusing. 
I have two suggestions:
1) Split it such that ``disabled = not request.user.has_perm(...)`` is in a different line and ``handle_permissions`` only manages the messages or
2) Rename to ``check_is_disabled`` or something like that